code/modules/mining/machine_vending.dm

```diff
@@ -51,6 +51,11 @@
 		new /datum/data/mining_equipment("Drone Ranged Upgrade",/obj/item/device/mine_bot_ugprade/cooldown,								600),
 		new /datum/data/mining_equipment("Drone AI Upgrade",	/obj/item/slimepotion/sentience/mining,									1000),
 		new /datum/data/mining_equipment("Jump Boots",			/obj/item/clothing/shoes/bhop,											2500),
+		new /datum/data/mining_equipment("Survival Knife",		/obj/item/weapon/kitchen/knife/combat/survival,							350),
+		new /datum/data/mining_equipment("Syringe Economy Pack",/obj/item/weapon/storage/box/medipens/survival,							2250),
+		new /datum/data/mining_equipment("Stimpack",			/obj/item/weapon/reagent_containers/hypospray/medipen/stimpack,			250),
+		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1250),
+		new /datum/data/mining_equipment("Whetstone",			/obj/item/weapon/sharpener,												400),
```

Comment:
Whetstones aren't a REALLY bad thing and I don't understand why they were removed anyways.